.gitlab-ci.yml

```diff
@@ -86,14 +86,30 @@ run-tests:
     - mkdir -p "build/$TARGET_DEVICE"
     - tools/docker/builder/openwrt/build.sh -v -d "$TARGET_DEVICE" -t "prplmesh-builder-$TARGET_DEVICE:$CI_COMMIT_SHORT_SHA-$CI_PIPELINE_ID" 2>&1 | tee "build/$TARGET_DEVICE/openwrt-build.log" | grep -E '^[[:blank:]]?make\[[12]\]|^Step|^ --->' --color=never --line-buffered
   after_script:
-    # if we're on master, tag the docker image with a persistant tag:
     - |
-      if [ $CI_COMMIT_REF_NAME == "master" ] ; then
-          echo "Keeping tag prplmesh-builder-$TARGET_DEVICE"
-          docker tag prplmesh-builder-$TARGET_DEVICE:$CI_COMMIT_SHORT_SHA-$CI_PIPELINE_ID prplmesh-builder-$TARGET_DEVICE
+      max_days=30
+      repository="prplmesh-builder-$TARGET_DEVICE"
```

Comment:
By using TARGET_DEVICE here, we will not automatically clean up after a device has been removed from CI (or renamed). So I think it would be better to use * here (if docker's filter supports that). Actually, maybe remove all old tags that start with prplmesh.

Reply:
Good catch, indeed, removing a target from CI becomes a problem (although a smaller problem).
I'll switch to `prplmesh-*` to include the prplmesh-builder images and prplmesh-tests-runner as well.
You're also right on the fact that it doesn't need to run every time. Given that it already takes a few seconds to run (even when there is no image to delete) and that we currently control every runner assigned to the project, I'm going to keep this on the runners.

I'll later add instructions to the wiki on how to use it.